models/members.js

```diff
@@ -37,6 +37,29 @@ const fetchMembers = async () => {
   }
 }
 
+/**
+ * changes the role of a new user to member
+ * @param userId { String }: User id of user to be modified
+ * @return existingMember { Boolean }: to show if user is already a member
+ */
+
+const moveToMembers = async (userId) => {
+  try {
+    const userDoc = await userModel.doc(userId).get()
+    const user = userDoc.data()
+    if (user.roles && user.roles.member) return true
+    const roles = user.roles ? { ...user.roles, member: true } : { member: true }
+    await userModel.doc(userId).update({
+      roles
+    })
+    return false
```

Comment:
The return statements seem a little bit confusing 🤔 
What if we return `{ isAlreadyMember: true }` on line 50?

And for the second case, we could return `{ movedToMember: true }`

What do you think?